.github/workflows/ci.yml

```diff
@@ -59,8 +59,8 @@ jobs:
           - make integration-test
           - TARGET=dev .github/smoke-test.sh
           - TARGET=prod .github/smoke-test.sh
-          - .github/analyse.sh
-          - .github/hypertest.sh
+          - TARGET=prod make api-validate
+          - TARGET=prod make api-test
```

Comment:
These didn't use `prod` before, but they probably should.

This doesn't work as need `matrix.target` set (probably refs https://github.com/libero/article-store/pull/272#discussion_r389700363).